### PR TITLE
add browser and os as parameters

### PR DESCRIPTION
**Pull request by adrian-moloca**

### CODE REVIEW

**Comment by matthewmayer** [comment]
we discussed before and there seemed to be consensus that the current userAgent implementation is excessively complex and hard to maintain - see https://github.com/faker-js/faker/pull/1761#issuecomment-1406431993 - i think this simplification should probably be prioritised over adding more parameters

**Comment by matthewmayer** [comment]
btw it looks like you accidentally remove `/dist` from .gitignore which has caused a large number of files to be added to this PR

File: .gitignore [hunk 1]
```diff
@@ -74,7 +74,6 @@ node_modules/
 versions.json
 
 # Dist
-/dist
```
Comment (ST-DDT) [inline .gitignore:77]: Please revert removing dist from the gitignore.

**Review by ST-DDT — CHANGES_REQUESTED** [review]

**Comment by ST-DDT** [comment]
**Team decision**

The PR looks broken and not updated in a long time.
If you are still interested please create a new feature request issue first.

**Pull request closed without merging**